src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -190,6 +193,59 @@ public function table(array $headers, array $rows)
         $this->newLine();
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
I think we need a new virtual `@method` on `StyleInterface` in order to make this true :)